.github/workflows/test.yml

```diff
@@ -171,11 +171,12 @@ jobs:
       - if: ${{ matrix.task == 'optimage'}}
         name: Test GPU IMAGE=2 ops
         run: GPU=1 IMAGE=2 python -m pytest -n=auto test/test_ops.py
-      - if: ${{ matrix.task == 'openpilot' }}
-        name: Test openpilot model compile and size
-        run: |
-          DEBUG=2 ALLOWED_KERNEL_COUNT=207 VALIDTEST=1 FLOAT16=1 DEBUGCL=1 GPU=1 IMAGE=2 python openpilot/compile2.py
-          python -c 'import os; assert os.path.getsize("/tmp/output.thneed") < 100_000_000'
+      # OpenCL needs cl_khr_fp16 to work
```

Comment:
Just cast it to a float after the read.

Reply:
I tried doing this https://github.com/tinygrad/tinygrad/pull/2264/commits/64e31ae8b5e89997b8359a74500181558f6c4546 which is the same as what exists in master, there will be no imageh actually rendered. This is what tinygrad master has:
https://github.com/tinygrad/tinygrad/blob/8ff2e135502170d55400dc45ea418b51c63b4e93/tinygrad/renderer/cstyle.py#L62
vs here:
https://github.com/tinygrad/tinygrad/blob/64e31ae8b5e89997b8359a74500181558f6c4546/tinygrad/renderer/cstyle.py#L62

so we can't really render imageh in CI if we wanna run FP16=1, because even casting that requires the extension, try compiling:
```py
from tinygrad.runtime.ops_gpu import compile_gpu
compile_gpu("""
__kernel void r_64_32_8_3_3_3_4_4_4(write_only image2d_t data0, read_only image2d_t data1, read_only image2d_t data2, const __global float* data3) {
    half4 val0 = read_imageh(data1, smp, (int2)(0, 0));
    float4 val1 = (float4)(val0.x,val0.y,val0.z,val0.w);
}
""") # program_source:4:49: error: loading directly from pointer to type '__private half' requires cl_khr_fp16. Use vector data load builtin functions instead
```

Reply:
Update: I removed the support of imageh from this PR for now, I can follow-up with another PR for this if you're interested in rendering imageh in kernels.